recipes/sofa/recipe.yaml

```diff
@@ -0,0 +1,185 @@
+context:
+  name: sofa
+  version: "25.06.00"
+  build_num: 0
+
+recipe:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/sofa-framework/sofa/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: a3a9345d71c3c62c26a0abddf18f7cbbd481bba5f095c92c0956f33530b50e69
+
+  patches:
+      # https://github.com/sofa-framework/sofa/pull/5664
+      - patches/0001-GUI-Fix-compilation-with-clang-and-boost-1.89-5664.patch
+      # https://github.com/sofa-framework/sofa/pull/5665
+      - patches/0002-Framework-Fix-compilation-with-clang20-5665.patch
+
+build:
+  number: ${{ build_num }}
+
+outputs:
+  - package:
+      name: libsofa
+    build:
+      script: scripts/build-lib
+      files:
+        - ${{ "Library/" if win }}lib/${{ "lib" if not win }}Sofa*
+        - ${{ "Library/" if win }}plugins/**/lib
+        - if: win
+          then:
+            - Library/bin/*
+            - Library/plugins/**/bin
+        # for copying activation/deactivation scripts
+        - etc/conda/*
+    requirements:
+      build:
+        - ${{ compiler('cxx') }}
+        - ${{ stdlib('c') }}
+        - cmake
+        - if: unix
+          then: make
+        - if: win
+          then: ninja
+      host:
+        - libboost-headers
+        - zlib
+        - tinyxml2
+        - eigen
+        - cxxopts
+        - gtest
+      run:
+        - zlib
```

Comment:
If this is required for `find_package(SOFA)` to succeed, perhaps it should be in `sofa-devel` ?